frontend/src/views/Pages/UserProfile/UserCard_FormUserData.vue

```diff
@@ -3,7 +3,7 @@
     <div>
       <b-row class="mb-4 text-right">
         <b-col class="text-right">
-          <a @click="showUserData ? (showUserData = !showUserData) : cancelEdit()">
+          <a href="#!" @click="showUserData ? (showUserData = !showUserData) : cancelEdit()">
```

Comment:
Das finde ich keine gut Lösung, da es nach dem Click in der URL sichtbar wird. Was ist mit `style="cursor: pointer;"`

Reply:
`http://localhost/vue/profile#!`

Reply:
Noch besser wäre es, wenn wir eine Klasse hätten die das Problem löst. Ich verstehe nicht, warum `class="cursor-pointer"` nicht funktioniert.

Reply:
cursor-pointer hat nicht direkt geholfen! Habe aber wie von @ulfgebhardt empfohlen folgendes hinzugefügt:
```
.cursor-pointer{
  cursor: pointer;
}
```